hphp/runtime/vm/jit/vasm-x64.cpp

```diff
@@ -1097,7 +1150,15 @@ void optimizeX64(Vunit& unit, const Abi& abi, bool regalloc) {
 
 void emitX64(Vunit& unit, Vtext& text, CGMeta& fixups,
              AsmInfo* asmInfo) {
-  vasm_emit<Vgen>(unit, text, fixups, asmInfo);
+#ifdef HAVE_LIBXED
+  if (RuntimeOption::EvalUseXedAssembler) {
+    vasm_emit<Vgen<XedAssembler>>(unit, text, fixups, asmInfo);
```

Comment:
You can return here and #endif to avoid duplicate lines 1157 and 1160.

Reply:
Addressed in https://github.com/facebook/hhvm/pull/8164/commits/ac4a93085f536b08c1f366528bb51a4599d8b290